scripts/installer/windows/product.wxs

```diff
@@ -85,37 +84,26 @@
           <Custom Action="Set_WIXUI_EXITDIALOGOPTIONALTEXT" After="CostFinalize">NOT Installed AND NOT WIX_UPGRADE_DETECTED</Custom>
         </InstallUISequence>
 
-        <CustomAction Id="InstallPromptFlowService"
-            Directory="DynamicCliDir"
-            Execute="commit"
-            Impersonate="no"
-            ExeCommand="cmd.exe /c SC CREATE PromptFlowService &quot;displayname=Prompt Flow Service&quot; start=auto type=userown binPath= &quot;\&quot;%ProgramFiles%\promptflow\pfsvc.bat\&quot;&quot;"
-            Return="check" />
         <CustomAction Id="StartPromptFlowService"
             Directory="DynamicCliDir"
             Execute="commit"
             ExeCommand="PowerShell.exe -WindowStyle hidden Start-Process -FilePath pfs -NoNewWindow -Wait"
             Return="asyncNoWait" />
-        <CustomAction Id="UninstallPromptFlowService"
```

Comment:
need to add back the auto start

Reply:
@wangchao1230  I think the key issue is not only whether to use admin.
But also can remove the per-user service when uninstall. Now the problem is that, becauseof the service， user can't uninstall the app.